queries/headers/all_recipients_bccd.yml

```diff
@@ -0,0 +1,8 @@
+name: "All recipients BCCd"
+type: "query"
+source: |
+  type.inbound
+  and length(recipients.to) == 0
+  and length(recipients.cc) == 0
+  and length(recipients.bcc) == 1
```

Comment:
`==` or `>=`?
or does it show up as a single "Undisclosed recipients"?

Reply:
believe it's a single one